src/metabase/lib/walk.cljc

```diff
@@ -139,3 +145,59 @@
    (fn [query path-type path stage-or-join]
      (when (= path-type :lib.walk/stage)
        (f query path stage-or-join)))))
+
+(mr/def ::path.stages-part
```

Comment:
Added some new schemas for the `lib.walk` path as we start using these tools more.